setup.py

```diff
@@ -13,9 +13,16 @@
 # governing permissions and limitations under the License.
 
 from setuptools import setup, find_packages
+import os
+
+v = open('haas/version.py')
+exec(v.read())
```

Comment:
@SahilTikale spent some time looking up how people generally do this in Python, and he says that this is actually one of the common methods.  @SahilTikale, could you link some sources you found?